[2022-08-22 20:15:01,956] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-22T20:10:00+00:00 [queued]>
[2022-08-22 20:15:02,119] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-22T20:10:00+00:00 [queued]>
[2022-08-22 20:15:02,119] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-22 20:15:02,119] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-22 20:15:02,119] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-22 20:15:02,232] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): popula_elastic_finalizados> on 2022-08-22 20:10:00+00:00
[2022-08-22 20:15:02,238] {standard_task_runner.py:52} INFO - Started process 98349 to run task
[2022-08-22 20:15:02,289] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'elastic_desk', 'popula_elastic_finalizados', 'scheduled__2022-08-22T20:10:00+00:00', '--job-id', '1930', '--raw', '--subdir', 'DAGS_FOLDER/etl_elastic_desk.py', '--cfg-path', '/tmp/tmpv718nbi4', '--error-file', '/tmp/tmpkjmov690']
[2022-08-22 20:15:02,290] {standard_task_runner.py:80} INFO - Job 1930: Subtask popula_elastic_finalizados
[2022-08-22 20:15:02,740] {task_command.py:371} INFO - Running <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-22T20:10:00+00:00 [running]> on host 1e1070d9e5ae
[2022-08-22 20:15:03,106] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clecio.antao@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=elastic_desk
AIRFLOW_CTX_TASK_ID=popula_elastic_finalizados
AIRFLOW_CTX_EXECUTION_DATE=2022-08-22T20:10:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-22T20:10:00+00:00
[2022-08-22 20:15:03,139] {base.py:271} INFO - GET http://192.168.2.100:9200/ [status:200 request:0.030s]
[2022-08-22 20:15:08,355] {base.py:271} INFO - HEAD http://192.168.2.100:9200/_template/desk_finalizados [status:200 request:0.014s]
[2022-08-22 20:15:08,779] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/es_pandas/es_pandas.py:58 RuntimeWarning: invalid value encountered in double_scalars
[2022-08-22 20:15:09,075] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.084s]
[2022-08-22 20:15:09,338] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.137s]
[2022-08-22 20:15:09,485] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.092s]
[2022-08-22 20:15:09,614] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.070s]
[2022-08-22 20:15:09,752] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.061s]
[2022-08-22 20:15:09,844] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.066s]
[2022-08-22 20:15:09,981] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.076s]
[2022-08-22 20:15:10,202] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.150s]
[2022-08-22 20:15:10,313] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.057s]
[2022-08-22 20:15:10,369] {logging_mixin.py:115} INFO - 0       22-08-2022
1       22-08-2022
2       22-08-2022
3       22-08-2022
4       22-08-2022
           ...    
4481    23-07-2022
4482    23-07-2022
4483    23-07-2022
4484    23-07-2022
4485    23-07-2022
Name: DataFinalizacao, Length: 4486, dtype: object
[2022-08-22 20:15:10,371] {python.py:173} INFO - Done. Returned value was: None
[2022-08-22 20:15:10,391] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=elastic_desk, task_id=popula_elastic_finalizados, execution_date=20220822T201000, start_date=20220822T201501, end_date=20220822T201510
[2022-08-22 20:15:10,494] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-22 20:15:10,598] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
